### PR TITLE
Pretty printing JSON Body

### PR DESCRIPTION
**Pull request by freezy-sk**

### CODE REVIEW

File: APIBlueprintGenerator.coffee [hunk 1]
```diff
@@ -13,7 +13,10 @@ APIBlueprintGenerator = ->
       return null
 
     headers = []
+    is_json = false
     for key, value of exchange.responseHeaders
+      if key == 'Content-Type'
+        is_json = (value.search(/(json)/i) > -1)
```
Comment (pksunkara) [inline APIBlueprintGenerator.coffee]: Please move these 2 lines under the next line.

File: APIBlueprintGenerator.coffee [hunk 2]
```diff
@@ -23,6 +26,8 @@ APIBlueprintGenerator = ->
     body = exchange.responseBody
     has_body = body.length > 0
     if has_body
+      if is_json
+        body = JSON.stringify(JSON.parse(body),null,4)
```
Comment (pksunkara) [inline APIBlueprintGenerator.coffee]: Please add spaces after commas.

File: APIBlueprintGenerator.coffee [hunk 3]
```diff
@@ -55,6 +63,8 @@ APIBlueprintGenerator = ->
     body = paw_request.body
     has_body = body.length > 0
     if has_body
+      if is_json
+        body = JSON.stringify(JSON.parse(body),null,4)
```
Comment (pksunkara) [inline APIBlueprintGenerator.coffee]: Please add spaces after commas.

File: APIBlueprintGenerator.coffee [hunk 4]
```diff
@@ -45,7 +50,10 @@ APIBlueprintGenerator = ->
   #
   @request = (paw_request) ->
     headers = []
+    is_json = false
     for key, value of paw_request.headers
+      if key == 'Content-Type'
+        is_json = (value.search(/(json)/i) > -1)
       if key in ['Content-Type']
```
Comment (pksunkara) [inline APIBlueprintGenerator.coffee]: The above 3 lines can be reduced to:

```coffee
      if key in ['Content-Type']
        is_json = (value.search(/(json)/i) > -1)
```

File: APIBlueprintGenerator.coffee (hunk 1 above)
Comment (freezy-sk) [inline APIBlueprintGenerator.coffee]: They can't be under next line because then they will be skipped

**Comment by pksunkara** [comment]
Review done. Please fix the above things and we can merge this.

File: APIBlueprintGenerator.coffee (hunk 1 above)
Comment (pksunkara) [inline APIBlueprintGenerator.coffee]: They will not be skipped because they will still be above `continue`.

Comment (pksunkara) [inline APIBlueprintGenerator.coffee]: I am asking them to be moved under the `if` condition below.

**Comment by freezy-sk** [comment]
done ;)

**Comment by pksunkara** [comment]
Thanks for the PR! :smile: :balloon:

**Comment by freezy-sk** [comment]
You're welcome :)

**Pull request merged**